test/unit/specs/components/Identity/createManualIdSyncs.spec.js

```diff
@@ -0,0 +1,94 @@
+import createCookieProxy from "../../../../../src/core/createCookieProxy";
```

Comment:
In general, I'd like to avoid making http requests to actual servers in tests.  This means the test will fail if the rlcdn.com domain goes down or stops accepting requests.  If you pulled out the processor creation to Identity/index.js, you could pass in a mock idSyncProcessor for this test.

Reply:
@jonsnyder I wouldn't feel it's a true test though. In the case with this code, if the domain stopped responding, It would just return the failed id sync in the failed array instead of the succeeded array. This doesn't affect my test.

Reply:
@jonsnyder Sorry, that will make the test fail I just realized. I'll work on a mock.

Reply:
@jonsnyder @Aaronius @jfkhoury from Aaron's Explanation of What a unit test is, It sounds like I shouldn't be testing idSyncs.process at all in createManualIdSyncs.spec.js and createIdSyncs.spec.js Because it's a subprocess in these files. So I guess I'll reserve idSyncs.process For future functional testing. I'll leave in the unit tests that actually apply.